python/servo/build_commands.py

```diff
@@ -324,6 +325,21 @@ def build(self, target=None, release=False, dev=False, jobs=None,
                     for ssl_lib in ["libcryptoMD.dll", "libsslMD.dll"]:
                         shutil.copy(path.join(env['OPENSSL_LIB_DIR'], "../bin" + msvc_x64, ssl_lib),
                                     servo_exe_dir)
+                    if os.environ["VisualStudioVersion"] == "14.0":
+                        vs_dir = os.environ["VS140COMNTOOLS"]
+                        vs_dll_dir = path.join(vs_dir, "..", "IDE", "Remote Debugger", os.environ["PLATFORM"].lower())
+                        for vs_dll in ["api-ms-win-crt-runtime-l1-1-0.dll"]:
```

Comment:
No need to make this a loop for just one item, we can always re-add the loop in the future if necessary.